File: Mozu.SiteBuilder.UX.Admin/Api/Models/Shipping/SiteShippingRegion.cs
using System.Runtime.Serialization;

namespace Mozu.SiteBuilder.UX.Admin.Api.Models.Shipping
{
    [DataContract]
    public class SiteShippingRegion
    {
        [DataMember(Name = "isoCountryCode")]
        public string ISOCountryCode { get; set; }
    }
}

File: Mozu.SiteBuilder.UX.Admin/Scripts/app/model/SiteShippingRegion.js
/**
 * @class Taco.model.SiteShippingRegion
 */
Ext.define('Taco.model.SiteShippingRegion', {
    extend: 'Taco.core.data.Model',
    fields: [
        { name: 'isoCountryCode', type: 'string' }
    ],
    idProperty: 'isoCountryCode',
    proxy: {
        type: 'ajaxproxy',
        api: {
            read: '/admin/app/shipping/regions/read',
            create: '/admin/app/shipping/regions/edit',
            update: '/admin/app/shipping/regions/edit',
            destroy: '/admin/app/shipping/regions/delete'
        },
        reader: {
            type: 'json',
            root: 'items',
            successProperty: 'success',
            messageProperty: "message"
        },
        writer: {
            allowSingle: false,
            type: 'json'
        }
    }
});